Feature/actions (#81)
cosmoscout/cosmoscout-vr: README.md

Many features are badly documented, it will crash without warning and may do other unexpected things.
We are working hard on improving the user experience - please [report all issues and suggestions](https://github.com/cosmoscout/cosmoscout-vr/issues) you have!_

For each release, [binary packages](https://github.com/cosmoscout/cosmoscout-vr/releases) are automatically created via [Github Actions](https://github.com/cosmoscout/cosmoscout-vr/actions).

When started for the very first time, some example datasets will be downloaded from the internet.
**This will take some time!**

> The progress of this operation is printed to the console window.

Author: ```suggestion
The progress of this operation is shown on the loading screen.
```
